docker/14-ngv.envsh

```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/env bash
+# This file will be sourced in the container before running the env substitution in the container.
+
+APP_NAME="${APP_NAME:-all}"
+
+APP_PATH="${APP_PATH:-/}"
+case "$APP_NAME" in
+"") ;;
+"/") ;;
+"all") ;;
+*) export APP_PATH="src/apps/${APP_NAME}" ;;
```

Comment:
It looks strange that the APP_PATH can finish, or not with a '/'.
Shouldn't it be empty by default?

Reply:
You're right, it's either empty or `src/apps/${APP_NAME}`